care/facility/tasks/redis_index.py

```diff
@@ -37,9 +44,9 @@ def load_redis_index():
             if load_static_data:
                 load_static_data()
         except ModuleNotFoundError:
-            logger.info("Module %s not found", module_path)
+            logger.debug("Module %s not found", module_path)
         except Exception as e:
-            logger.info("Error loading static data for %s: %s", plug.name, e)
+            logger.exception("Error loading static data for %s: %s", plug.name, e)
```

Comment:
```suggestion
            logger.error("Error loading static data for %s: %s", plug.name, e)
```

I meant to say logger.error in the previous review